packages/kestrel_datasource_stixshifter/src/kestrel_datasource_stixshifter/worker/utils.py

```diff
@@ -30,3 +32,19 @@ class TranslationResult:
     success: bool
     data: Union[None, dict, DataFrame]
     log: Optional[WorkerLog]
+
+
+def disable_cert_verification_on_transmission(trans: StixTransmission):
+    ot = trans.entry_point.transmission()
+
+    # currently all the following attributes point to the same object
+    # iterate through them in case stix-shifter code changes in the future
+    for eps in [
+        "_BaseEntryPoint__ping_connector",
+        "_BaseEntryPoint__query_connector",
+        "_BaseEntryPoint__results_connector",
+        "_BaseEntryPoint__status_connector",
```

Comment:
Do we need the delete connector in here too?

Reply:
Good point! Rules generalized for enumerating the objects to patch.